docs/install/any-kubernetes-cluster.md

```diff
@@ -385,6 +385,10 @@ At this point, you have a basic installation of Knative Serving!
 
 ### Optional Serving extensions
 
+You can apply extensions to customize your Knative Serving installation.
+For more information about the extensions you can apply, see
+[Knative installation files and dependencies](./custom-install/custom-install.md#knative-serving-installation-files).
```

Comment:
I'd remove dependencies here and maybe call it something like extensions instead, it reads a bit confusing to me that they're extensions but then it says _Knative installation files and dependencies_ - this makes it feel like core components rather than options

Reply:
Title updated.